dmci/mmd_tools/check_mmd.py

```diff
@@ -174,12 +192,12 @@ def full_check(doc): # pragma: no cover
     urls = [elem.text for elem in url_elements]
     if len(urls) > 0:
         logger.debug('Checking element(s) containing URL ...')
-        urls_ok = check_urls(urls)
+        urls_ok = check_urls(urls, allow_no_path=True)
```

Comment:
Any reason you decided to allow no path?

Reply:
Because the tests failed, but we should maybe check what we want to do here.